scripts/repoHandlers.js

```diff
@@ -71,41 +71,61 @@ function updateTrackedReposInBitbucketOrg(trackedRepos, websiteUrl) {
  * @param {string} websiteUrl - The URL of the website.
  * @param {boolean} isOrg - Whether github organization or user.
  */
+
+
```

Comment:
This space might stop the IDE from understanding the structure of this function.